source/TimePicker.js

```diff
@@ -585,14 +625,33 @@
 		meridiemPickerChanged: function (sender, event) {
 			if(this.syncingPickers) return true;
 
-			var hour = this.$.hour.get('value'),
-				value = event.value;
+			var meridiemPicker = event.originator,
+				meridiems = meridiemPicker.get('meridiems'),
+				meridiem = meridiems[meridiemPicker.get('value')],
+				start = meridiem.start.split(':'),
+				startHour = parseInt(start[0], 10),
+				offset = meridiemPicker.get('offset'),
```

Comment:
Similar to above, we should consider documenting the `offset` property of `MeridiemPicker` and/ or just use direct property access i.e. `meridiemPicker.offset`.